README.md

```diff
@@ -152,11 +151,11 @@ Next is to run a few DL models to validate the end-to-end functionality.
     ```
     cd examples/resnet
```

Comment:
Good catch. Done.

Reply:
I have removed the resnet example because the pre-trained model that's available from the internet won't quite work with our inference script due to different data format (NCHW vs NHWC).